pkgs/test/texlive/default.nix

```diff
@@ -105,4 +105,16 @@
     mkdir "$out"
     mv document*.svg "$out"/
   '';
+
+  # the language files generated in scheme-full must be identical to the pristine copies in hyphen-base
+  hyphen-base = runCommandNoCC "texlive-test-hyphen-base" {
+    schemeFull = texlive.combined.scheme-full;
+    hyphenBase = lib.head texlive.hyphen-base.pkgs;
+  } ''
+    mkdir "$out"
+
+    for fname in language.{dat,def,dat.lua} ; do
+      diff -u {"$hyphenBase","$schemeFull"/share/texmf}/tex/generic/config/"$fname" | tee "$out/$fname.patch"
```

Comment:
This may not be obvious from the PR: the generated language files will now be identical to the originals in `scheme-full` (i.e. I got rid of the duplicated `% from` lines created by the old `hyphens.sed`). So I have added a test, after seeing how @veprbl added a test for `texdoc`. A test seems like a good idea, a breaking change in the language files might go undetected otherwise.

Reply:
I have updated the tests to check `scheme-infraonly` as well. Now they are more likely to fail if a future update requires changes to `hyphens.sed`.